### DISCUSSION TITLE
Is it OK to ignore multi-axis fallback names? Or should them be also documented in the GFAxisRegistry?

### DISCUSSION THREAD

**Discussion by felipesanches** [fonttools/fontbakery#3129] [Q&A] [answered]
According to the OpenType spec, the [STAT table](https://docs.microsoft.com/en-us/typography/opentype/spec/stat) can have **Format 4** axis value tables which serve the purpose of giving a name to a combination of coordinates in some of the font variation axes.

Here's an example of a font that has format-4 tables:

![Screenshot from 2020-12-09 06-08-57](https://user-images.githubusercontent.com/213676/101608889-1060b780-39e5-11eb-8879-f6cdfa1078f8.png)

I'd be glad to hear from anyone who's got an opinion on this, of course. But I'm specially interested in hearing from @vv-monsalve, @davelab6, @RosaWagner, @m4rc1e or anyone else involved in building the GF Axis Registry.

### Note:
This is related to the following recent bugfix:
  - **[com.google.fonts/check/STAT/gf-axisregistry]:** Ignore format-4 entries on STAT table because the GF Axis Registry does not list any multi-axis fallback name which is what such entries are designed to describe (issue #3106)

**Comment by m4rc1e** [comment, marked as answer]
Currently, the GF Axis Registry doesn't define any format 4 axis values. We only define axis values for individual axes only (formats 1,2,3).

Perhaps in the future we may find certain axis combinations work well together and only then will we require format 4. I reckon we'll discover these combinations in families with tonnes of axes made by the likes of Type Network.

Imo, skipping format 4 axis values for the time being is fine.

**Reply by RosaWagner** [reply]
Lexend could need it and Commissioner would have had needed it too though
